### ISSUE TITLE
Implement functionality of clicking on activeNavigation button

### ISSUE THREAD

**Issue by i315315** [amosproj/amos2022ws02-automotive-test-app#41]
## User story
1. As a User
2. I want to be able to switch back to the navigation use case that is currently active by clicking on the activeNavigation button
3. So that I can easily see the current status of the navigation and possibly change it if wanted.

With this User story, the functionality of clicking on the activeNavigation button is to be implemented. when clicking on the activeNavigation button, the user should be led back to the navigation use case that is currently active.

## Acceptance criteria
* It is possible to click on the activeNavigation button.
* When clicking on the activeNavigation button, the user is led to the navigation use case that is currently active.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
